code/src/Components/Summary.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Summary =({personInput, destinationInput,budgetInput,awayInput}) => {
+    return (
+        <main className='main-container summary'>
+            <h1 tabindex="0">This survey show that:</h1>
+            <p tabindex="0"><span>You want to travel {personInput}</span></p> 
```

Comment:
I have not managed to figure it out but, {personInput} is not displayed in the summary